setup.py

```diff
@@ -3,31 +3,39 @@
 with open('requirements.txt') as f:
     requirements = f.readlines() 
 
-long_description = '''Python wrapper package for NASA GIBS API \ 
-    with additional utility functions to prepare downloaded images for \
+long_description = '''Python package for downloading NASA GIBS imagery \ 
+    with additional functionality to prepare downloaded images for \
     machine learning pipeline''' 
 
 setup(
-    name ='GIBSDownloader', 
-    version ='1.0.0', 
-    author ='Fernando Lisboa, Shivam Verma', 
-    author_email ='0fernando.lisboa@gmail.com', 
-    url ='https://github.com/spaceml-org/NASA-GIBS-Downloader', 
-    description ='Downloading tool for NASA GIBS', 
-    long_description = long_description, 
-    long_description_content_type ="text/markdown",  
-    packages = find_packages(), 
-    entry_points ={ 
+    name='GIBSDownloader', 
+    version='1.0.0', 
+    license='Apache License 2.0',
+    author='Fernando Lisboa, Navya Sandadi, Shivam Verma', 
+    author_email='0fernando.lisboa@gmail.com', 
```

Comment:
I'd be concerned about getting spam email if a bot were to crawl your email address from here.